### ISSUE TITLE
Can this stack be deployed?

### ISSUE THREAD

**Issue by stickystyle** [ChatAndSlash/Slashbot#1]
This looks like an awesome project, and is something I'd like to run on my company slack instance. 
But before I sink any time into getting it setup, I wanted to check in to see if all the parts are currently available to get it running privately as you have this comment on the readme...
"Use DM to deploy to live servers. See that project's README. NOTE: DM is not yet publicly available."

Bummed I missed out when it was published, is there any technical reason why you pulled it?

**Comment by DanHulton** [comment]
DM will likely not be made publicly available - it was a collection of Ansible scripts that relied on a whole bunch of secrets and configuration values that I wouldn't publish, so it wouldn't work out-of-the box.

Further complicating matters is that you'd need to set up a Slack developer account, generate a Slack app, set up a URL for your app to live at that has working, valid SSL certificates... It's a whole deal.

_Even further_ complicating matters is the fact that the reason I pulled the app down is that it requires a rewrite to work with Slack's most-recent change for team and user authorization, and I never had the time to do the rewrite.

In short, there's a whole bunch of work required to get this code in working condition again.  If you have the time, energy, and drive to do the rewrite, you're more than welcome to fork the project and make the changes.  But I didn't and don't have the time to fix it up again, so the best I can do is wish you good luck.  =)
